test/blackbox-tests/test-cases/melange/public-library-with-runtime-deps.t

```diff
@@ -0,0 +1,94 @@
+Test simple interactions between melange.emit and copy_files
+
+  $ mkdir prefix
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (package (name foo))
+  > (using melange 0.1)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (alias mel)
+  >  (target output)
+  >  (libraries foo)
+  >  (runtime_deps assets/file.txt))
+  > EOF
+
+  $ mkdir -p lib/nested
+  $ echo "Some text" > lib/index.txt
+  $ echo "Some nested text" > lib/nested/hello.txt
+  $ cat > lib/dune <<EOF
+  > (library
+  >  (public_name foo)
+  >  (modes melange)
+  >  (melange.runtime_deps index.txt nested/hello.txt))
+  > EOF
+  $ cat > lib/foo.ml <<EOF
+  > let dirname = [%bs.raw "__dirname"]
+  > let () = Js.log2 "dirname:" dirname
+  > let file_path = "./index.txt"
+  > let read_asset () = Node.Fs.readFileSync (dirname ^ "/" ^ file_path) \`utf8
+  > EOF
+
+  $ dune build --display=short
```

Comment:
As a general rule, we should avoid the `--display=short` whenever possible in tests. It usually just adds noise.